sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -47,22 +50,72 @@ -(void)testInit
     XCTAssertNotNil(self.store, @"store creation failed");
 }
 
+// Tests MSCoreDataStore Read, Update and Delete operations are case insensitive for the Id column.
+-(void)testReadUpdateDelete_CaseInsensitiveIdOperations
+{
+    NSError *error;
+
+    // Seed data to be inserted in the table
+    NSString * const originalItemId = @"itemid";
+    NSString * const originalText = @"original text";
```

Comment:
The same strings are used in multiple places. Repeating string literals is a potential maintenance pain. Are there advantages of not defining the variables once and reusing multiple times?